packages/cheqd/tests/cheqd-did-registrar.e2e.test.ts

```diff
@@ -188,6 +190,8 @@ describe('Cheqd DID registrar', () => {
       method: 'cheqd',
       didDocument: new DidDocumentBuilder(did)
         .addContext(SECURITY_JWS_CONTEXT_URL)
+        .addController(did)
```

Comment:
Not related to this PR, but it would still be great if we can look at what the issues are with P256 keys